pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md

```diff
@@ -156,7 +158,8 @@ Indicates that the Faas hook could not be invoked.
 #### Reasons
 Implementation errors inside Faas hook.
 #### Solutions
-Double-check and test the Faas Hook code.
+Double-check and test the Faas Hook code. You can refer to the [Faas Documentation](liveperson-functions-deployment.html#testing-your-function) to find out how test your function code.
```

Comment:
find out how test your function code -> find out how **to** test your function code